src/FormatCommand.cs

```diff
@@ -49,6 +50,10 @@ internal static RootCommand CreateCommandLineOptions()
                 {
                     Argument = new Argument<string?>("severity") { Arity = ArgumentArity.ZeroOrOne }.FromAmong(SeverityLevels)
                 },
+                new Option(new[] { "--diagnostics" }, Resources.A_space_separated_list_of_diagnostic_ids_to_use_as_a_filter_when_fixing_code_style_or_3rd_party_issues)
+                {
+                    Argument = new Argument<string[]>(() => Array.Empty<string>())
```

Comment:
I would like it if we could provide a helpful message if this option is provide and we are not in a `--fix-stlye` or `--fix-analyzers` mode